spec/lib/whitehall_importer/change_history_spec.rb

```diff
@@ -0,0 +1,81 @@
+RSpec.describe WhitehallImporter::ChangeHistory do
+  let(:first_edition) do
+    build(:whitehall_export_edition,
+          state: "superseded",
+          revision_history: [build(:whitehall_export_revision_history_event,
+                                   state: "published",
+                                   created_at: Time.zone.now.last_year.rfc3339)])
+  end
+
+  it "returns change history for the published edition" do
+    superseded_edition = build(:whitehall_export_edition,
+                               state: "superseded",
+                               change_note: "Some fantastic changes",
+                               revision_history: [build(:whitehall_export_revision_history_event,
+                                                        state: "published",
+                                                        created_at: Time.zone.now.beginning_of_day.rfc3339)])
+    published_edition = build(:whitehall_export_edition,
+                              state: "published",
+                              change_note: "Some great changes",
+                              revision_history: [build(:whitehall_export_revision_history_event,
+                                                       state: "published")])
+    whitehall_export = build(:whitehall_export_document,
+                             editions: [first_edition, superseded_edition, published_edition])
```

Comment:
I don't think creating this published edition actually has any impact on the test?

Reply:
Nope it doesn't, since however I have updated the test so it makes use of the published edition...